asserts/account_key.go

```diff
@@ -212,13 +212,13 @@ func (akr *AccountKeyRequest) Until() time.Time {
 	return akr.until
 }
 
-// PublicKey returns the underlying public key of the requested account key.
-func (akr *AccountKeyRequest) PublicKey() PublicKey {
```

Comment:
Do we want to lose this method? It's a core header so seems reasonable to have an API to get to it ... but maybe it's just not that useful?